### DISCUSSION TITLE
Faculdade ou cursos?

### DISCUSSION THREAD

**Discussion by jessicasilvano** [frontendbr/forum#1101] [Discussão]
Olá, estou no último ano do curso técnico de informática e agora estou planejando o que fazer ano que vem. Ainda não me sinto pronta para começar num emprego na área de programação, por isso tenho pensado em me aprofundar nas coisas que aprendi no curso técnico (java, programação mobile, programação web) ao invés de começar numa faculdade. No entanto, tenho medo de ficar perdida sobre o que aprender primeiro. Além disso, eu tenho medo de quanto mais eu atrasar a ida pra faculdade, mais difícil ficar para eu arrumar um emprego.

O que vocês fariam no meu lugar? Abandonaria uma facul pra fazer cursos ou investiriam numa boa faculdade?

**Comment by expalmer** [comment]
1) Faz faculdade, nem que seja 1 cadeira. => para poder pegar um estágio
2) Faz cursos online no codeacademy e afins...
3) Fica tentando entrar num estágio na área de programação.

**Comment by laurindo** [comment]
@Jess710 Se você mora no Brasil e deseja um dia imigrar, ou conseguir emprego fora pra trabalhar na área, a faculdade vai ser importante. Além de também você poder fazer uma pós, mestrado etc. 

Eu faria a faculdade e ia estudando por fora, criando projetos, praticando, fazendo cursos da udemy por exemplo.

A minha namorada está na mesma situação, mas a dica que dei foi essa q falei pra você. Ela já está participando de algumas entrevistas.

**Comment by tarciozemel** [comment]
Abandonaria uma facul pra fazer cursos.

**Comment by bizarro** [comment]
O único cenário que eu acho válido não fazer faculdade é caso financeiramente esteja pesando muito para o seu orçamento ou da sua família, pois realmente não é tão fácil desembolsar uma grana todo o mês para terminar a graduação.

Tirando isso, acho que é sempre válido fazer, principalmente para quem tem interesse em trabalhar fora do país e não possui uma naciondalidade sem ser a brasileira. Geralmente as empresas perguntam se você tem algum degree relacionado na área de tecnologia, pois o processo de vistos e imigração é sempre menos burocrático para profissionais que tem um diploma.

Agora, um ponto importante é que talvez você acabe não aprendendo absolutamente nada de novo na faculdade, pois a maioria tem uma grade desatualizada e totalmente fora do que está sendo aplicado nas empresas, então por isso é sempre melhor estudar por fora mesmo. :joy:

Resumão: faça faculdade, estude por fora e trabalhe na área o quanto antes.

**Comment by kvnol** [comment]
Cursos.

**Comment by rsmelo92** [comment]
Faculdade é estresse puro durante 4 ou 5 anos. Mas vale a pena, vc aprofunda seu conhecimento e faz um network que em outro lugar seria impossível. Colegas e professores podem abrir portas que podem te fazer a diferença, e vc pode mais tarde abrir portas para outros colegas. Se no momento vc tem condições, vale muito a pena.

**Comment by erisonnicodemos** [comment]
Se possível faça os dois, na Udemy tem muito cursos baratos e legais, sobre a faculdade acho importante muitas empresas dão valor ao diploma, principalmente muti-nacionais.

**Comment by kivervinicius** [comment]
A faculdade é algo que irá te proporcionar uma experiência incrível, algo que em um curso não irá, não estou dizendo a nível de conhecimento, pois na faculdade você terá uma base teórica e um pouco de prática, cursos vieram para complementar uma área que você se interessa, um complementa o outro, cursos são muito interessantes porém faculdade te dá um networking incrível. 

PS: Sou professor universitário e também ministro cursos, são coisas que sou apaixonado, vai por mim você vai gostar.

**Comment by danilocandido** [comment]
Tem faculdade pública também pra estudar de graça. 
Basta fazer o ENEM, tem FIES, tem PROUNI.

**Comment by caiomyrapereira** [comment]
eu tbm to perdido  kkkk

**Comment by felipefialho** [comment]
Linkando: 

#809 
#936

**Comment by thismarcoantonio** [comment]
Acredito que existem casos e casos, eu particularmente não sou muito chegado a faculdade, grande parte por coisas que aconteceram em minha vida durante ela (comecei na federal, tive que largar no primeiro periodo por não ter dinheiro para pagar transporte).

Após ter perdido esta oportunidade, comecei a estudar todo dia 6 a 8 horas por dia, desde o começo. Hoje trabalho com desenvolvimento web, sem nenhum certificado ou diploma, acredito que grande parte das empresas e startups priorizam o conhecimento do que um diploma.

Quanto a morar fora do pais, pode sim ser útil, mas venho recebendo ofertas de fora também, sem mesmo ter nada...

**Comment by laurindo** [comment]
No início da minha carreira, foi graças a faculdade que comecei meu estudos. 

Depois fui observando que mesmo sem uma faculdade eu conseguiria bons empregos, pois o que importava era o conhecimento, experiência, código no github, portfólio, etc ; então resolvi trancar a faculdade, passei uns 4 anos parados, mas depois comecei a receber propostas no LinkedIn pra trabalhar na Europa, Canadá, Austrália, etc. 

A maioria dos países q tinham interesse sempre perguntaram se eu tinha formação em alguma aérea, pois com o certificado da faculdade facilitava a emissão do visto, com excessao da Holanda q não exige tanto a questão do certificado, mas mesmo assim sempre perguntam , entao novamente percebi o qnt era importante uma graduação e resolvi voltar pra me formar, pois ainda tenho vontade de trabalhar fora. 

Por isso que se você tem vontade de um dia imigrar, a formação é requisito obrigatório. 

Acredito que faculdade é importante, curso é importante, portfólio, projetos no github tbm são,e o mais importante de todos é o inglês. 

Indico passar por essas etapas.
